content/de/docs/concepts/workloads/pods/_index.md

```diff
@@ -0,0 +1,373 @@
+verwenden, 
+einen für jede Instanz. In Kubernetes wird dies typischerweise als Replikation
+bezeichnet.
+Replizierte Pods werden normalerweise als eine Gruppe durch eine 
+Workload-Ressource und deren 
+{{<glossary_tooltip text = "Controller" term_id = "controller">}} erstellt 
+und verwaltet.
+
+Die Seite [Pods und Controller](#pods-and-controllers) beschreibt, wie 
+Kubernetes Workload-Ressourcen und deren Controller verwendet, um Anwendungen 
+zu skalieren und zu heilen.
+
+### Wie Pods mehrere Container verwalten
+
+Pods unterstützen mehrere kooperierende Prozesse (als Container), die eine 
+zusammenhängende Serviceeinheit bilden. Kubernetes plant und stellt automatisch 
+sicher, dass sich die Container in einem Pod  auf demselben physischen oder 
+virtuellen Server im Cluster befinden. Die Container können Ressourcen und
+Abhängigkeiten gemeinsam nutzen, miteinander kommunizieren und
+ferner koordinieren wann und wie sie beendet werden.
+
+Zum Beispiel könnten Sie einen Container haben, der als Webserver für Dateien in
+einem gemeinsamen Volume arbeitet. Und ein separater "Sidecar" -Container 
+aktualisiert die Daten von einer externen Datenquelle, siehe folgenden 
+Abbildung:
+
+{{< figure src="/images/docs/pod.svg" alt="example pod diagram" width="50%" >}}
+
+Einige Pods haben sowohl {{<glossary_tooltip text = "Initialisierungs-Container" 
+term_id = "init-container">}} als auch {{<glossary_tooltip text = 
+"Anwendungs-Container" term_id = "app-container">}}. 
+Initialisierungs-Container werden gestartet und beendet bevor die 
+Anwendungs-Container gestartet werden.
+
+Pods stellen standardmäßig zwei Arten von gemeinsam Ressourcen für die 
+enthaltenen Container bereit:
+[Netzwerk](#pod-networking) und [Speicher](#pod-storage).
+
+
+## Mit Pods arbeiten
+
+Sie werden selten einzelne Pods direkt in Kubernetes erstellen, selbst 
+Singleton-Pods. Das liegt daran, dass Pods als relativ kurzlebige 
+Einweg-Einheiten konzipiert sind. Wann Ein Pod erstellt wird (entweder direkt 
+von Ihnen oder indirekt von einem
+{{<glossary_tooltip text = "Controller" term_id = "controller">}}), wird die 
+Ausführung auf einem {{<glossary_tooltip term_id = "node">}} in Ihrem Cluster 
+geplant. Der Pod bleibt auf diesem (virtuellen) Server, bis entweder der Pod die
+Ausführung beendet hat, das Pod-Objekt gelöscht wird, der Pod aufgrund 
+mangelnder Ressourcen *evakuiert* wird oder oder der Node ausfällt. 
+
+{{<note>}}
+Das Neustarten eines Containers in einem Pod sollte nicht mit dem Neustarten 
+eines Pods verwechselt werden. Ein Pod ist kein Prozess, sondern eine Umgebung 
+zur Ausführung von Containern. Ein Pod bleibt bestehen bis er gelöscht wird.
+{{</ note>}}
+
+Stellen Sie beim Erstellen des Manifests für ein Pod-Objekt sicher, dass der 
+angegebene Name ein gültiger
+[DNS-Subdomain-Name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)
+ist.
+
+### Pods und Controller
```

Comment:
Because there is a link to this, you might want to write:
```suggestion
### Pods und Controller {#pods-and-controllers}
```
or change the link on line 102.

Reply:
did the latter, i.e changed the link